theano/sandbox/gpuarray/type.py

```diff
@@ -95,6 +140,21 @@ def __init__(self, dtype, broadcastable, context_name=None, name=None):
                             (self.__class__.__name__, self.dtype))
 
     def clone(self, dtype=None, broadcastable=None):
+        """
```

Comment:
I agree, but the documentation does not propagate for reimplemented functions.  I could just drop the docstrings for those and sphinx show the base classes, however I personally dislike having to play a hide and seek with documentation just to find what a certain class can do.

We could however automate the transmission of docstrings for certain methods, which could alleviate this problem.